analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -649,6 +663,34 @@ def parse_convert_reports(input_dirs: List[str],
             number_of_reports
 
 
+def _generate_json_output(
+        severity_map: Dict, input_dirs: List[str], output_type: str,
+        output_path: str, trim_path_prefixes: Optional[List[str]],
+        skip_handler: Callable[[str], bool]) -> int:
```

Comment:
It will be easier to see what is the return type and where the function definition ends:
```suggestion
        skip_handler: Callable[[str], bool]
) -> int:
```